example/CMakeLists.txt

```diff
@@ -24,13 +27,13 @@ if (WIN32)
   set(ADDITIONAL_LINK_LIBRARIES ws2_32)
 endif()
 link_libraries(OpenSSL::Crypto OpenSSL::SSL ${ADDITIONAL_LINK_LIBRARIES})
-add_subdirectory(src/websocket_market_data_simple)
-add_subdirectory(src/websocket_market_data_trade)
-add_subdirectory(src/websocket_market_data_advanced)
+#add_subdirectory(src/websocket_market_data_simple)
+#add_subdirectory(src/websocket_market_data_trade)
+#add_subdirectory(src/websocket_market_data_advanced)
 add_subdirectory(src/websocket_execution_management)
-add_subdirectory(src/rest_execution_management_simple)
-add_subdirectory(src/rest_execution_management_complex)
-add_subdirectory(src/market_making)
-add_subdirectory(src/cross_exchange_arbitrage)
-add_subdirectory(src/enable_library_logging)
-add_subdirectory(src/rest_custom_service_class)
+#add_subdirectory(src/rest_execution_management_simple)
+#add_subdirectory(src/rest_execution_management_complex)
+#add_subdirectory(src/market_making)
+#add_subdirectory(src/cross_exchange_arbitrage)
+#add_subdirectory(src/enable_library_logging)
+#add_subdirectory(src/rest_custom_service_class)
```

Comment:
If you'd like to only build one "subdirectory", then it can be achieved by invoking "make websocket_execution_management" on the commandline.